crates/flox/doc/flox-run.md

```diff
@@ -11,21 +11,58 @@ flox-run - run app from current project
 
 # SYNOPSIS
 
-flox [ `<general-options>` ] run [ `<options>` ] [ -- [ `<command args>` ... ] ]
+flox [ `<general-options>` ] run [ `<run-options>` ] [ -- [ `<command args>` ... ] ]
 
 # DESCRIPTION
 
 Run flake application from the requested package (or "installable").
```

Comment:
An application from a flake is an installable, right? Not sure exactly what would make that more clear, maybe:
Run an "installable," an application from a flake.

Reply:
Does `nix run` not only look at the `app` flake output? Otherwise, we could word it as follows.
```suggestion
Run an application from the `app` output attribute.
```

Reply:
No it looks at `legacyPackages` and `packages` as well. Honestly I think if we just give a high-level description that's probably best. So that users don't have to understand flakes, outputs, apps vs packages, etc